guides/v2.2/cloud/project/project-deploy-into-subfolder.md

```diff
@@ -0,0 +1,80 @@
+---
+group: cloud-guide
+title: Deploy Magento into sub-folder
+functional_areas:
+  - Cloud
+  - Configuration
+  - Setup 
+---
+
+Use the following example of configuration if you deploy Magento in the not a root folder but in a subfolder - for example to `magento_src`:
+
+```
+docs
+docker
+config
+magento_src
+   app
+   generated
```

Comment:
I would remove `generated` mention from example, so people don't have ab impression that this should go under git.

Reply:
replaced with `bin` ;)

Thank you, @YPyltiai